drizzle.config.ts

```
@@ -2,11 +1,12 @@
 import { defineConfig } from 'drizzle-kit'
+import 'dotenv/config'
 
 export default defineConfig({
   dialect: 'turso',
   schema: './src/api/db/schema.ts',
   out: './migrations',
   dbCredentials: {
-    url: process.env.TURSO_DATABASE_URL!,
-    authToken: process.env.TURSO_AUTH_TOKEN,
+    url: process.env.TURSO_DEV_DATABASE_URL!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)